internal/lsp/server.go

```diff
@@ -1631,7 +1626,25 @@ func (l *LanguageServer) handleTextDocumentCodeLens(
 		return nil, nil // return a null response, as per the spec
 	}
 
-	return rego.CodeLenses(ctx, params.TextDocument.URI, contents, module) //nolint:wrapcheck
+	ls, err := rego.CodeLenses(ctx, params.TextDocument.URI, contents, module)
+	if err != nil {
+		return nil, err
```

Comment:
Just use the `//nolint:wrapcheck` comment here if you don't think wrapping the error adds anything.